ykrt/src/mt.rs

```diff
@@ -629,57 +649,73 @@ impl Drop for MT {
     }
 }
 
+/// [MTThread]'s major job is to record what state in the "interpreting/tracing/executing"
+/// state-machine this thread is in. This enum contains the states.
+enum MTThreadState {
+    /// This thread is executing in the normal interpreter: it is not executing a trace or
+    /// recording a trace.
+    Interpreting,
```

Comment:
The name "Interpreting" might be a bit confusing in that it appears to suggest that this thread is interpreting something, when in fact the parent thread is interpreting and this thread is really doing nothing, right?

Reply:
It means this thread is doing something that isn't a) tracing or b) executing a trace. "Interpreting" was the best term I could think of for that "something" (so far, at least).

Reply:
Yeah, just wanted to clear that up. I don't have a better suggestion at the moment. Maybe "Idling"?